src/sdam/srv_polling.ts

```diff
@@ -112,9 +109,8 @@ export class SrvPoller extends TypedEventEmitter<SrvPollerEvents> {
     }
 
     this._timeout = setTimeout(() => {
-      this._poll().catch(unexpectedRuntimeError => {
-        this.logger.error(`Unexpected ${new MongoRuntimeError(unexpectedRuntimeError).stack}`);
-      });
+      // eslint-disable-next-line @typescript-eslint/no-empty-function, @typescript-eslint/no-unused-vars
```

Comment:
I assume this, `failure()` and `parentDomainMismatch()` are left here as placeholders to plug the new logger into? If so, can we add comments here, line 124 and line 130 with the JIRA ticket so we know why we're not refactoring/removing these?

Reply:
I was reading the JIRA ticket again and we should rip these out entirely. Sorry about that - so let's remove all these comments and empty methods.

Reply:
Just to clarify, for `schedule`, are we removing the entire `schedule` method or just the empty function provided to the `catch` case on the call to `this._poll`?

Reply:
I think actually we can leave the empty function here to avoid an unhandled promise rejection.